Tests/Marketplace/marketplace_services.py

```diff
@@ -1023,14 +1040,14 @@ def upload_to_storage(self, zip_pack_path, latest_version, storage_bucket, overr
                 zip_to_upload_full_path = overridden_upload_path
             else:
                 version_pack_path = os.path.join(storage_base_path, self._pack_name, latest_version)
-                existing_files = [f.name for f in storage_bucket.list_blobs(prefix=version_pack_path)]
+                existing_files = [Path(f.name).name for f in storage_bucket.list_blobs(prefix=version_pack_path)]
```

Comment:
?

Reply:
`f.name` is the full path, `Path(f.name).name` is the files name